backend/src/routers/accountsRouter.ts

```diff
@@ -65,6 +65,16 @@ accountsRouter.get('/info', handleAsync(async (req, res) => { // TODO make this
   res.send(result);
 }));
 
+accountsRouter.delete('/delete', handleAsync(async (req, res) => {
+  const uuid = await userUuidFromReq(req);
```

Comment:
this is fine, but I should note you could also use `authenticateUUID` for this purpose, since `authenticateUUID` automatically throws a 403 if the token is invalid.